widgetastic_patternfly.py

```diff
@@ -618,7 +618,7 @@ def select_by_visible_text(self, *items):
                 './div/ul/li/a[./span[contains(@class, "text") and normalize-space(.)={}]]'.format(
                     quote(text)),
                 parent=self)
-        self.close()
+        # self.close()
```

Comment:
This is for @mfalesni to check . The self.select_item_type.select_by_visible_text(item_type) at line 99 in catalog_item.py works fine If I comment this line and gives error http://pastebin.test.redhat.com/462742 with the line uncommented.